### PR TITLE
UIAC-39 Settings (Acquisition units) | Apply baseline keyboard shortcuts

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://issues.folio.org/browse/UIAC-39
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/settings/AcquisitionUnits/AcquisitionUnitDetails/AcquisitionUnitDetails.js [hunk 1]
```diff
@@ -11,37 +13,27 @@ import {
   KeyValue,
   Pane,
   Row,
+  HasCommand,
+  checkScope,
 } from '@folio/stripes/components';
 import { ViewMetaData } from '@folio/stripes/smart-components';
+import {
+  useAccordionToggle,
```
Comment (AndreiShumski2) [inline src/settings/AcquisitionUnits/AcquisitionUnitDetails/AcquisitionUnitDetails.js:21]: Try to use `AccordionStatus` instead.

File: src/settings/AcquisitionUnits/AcquisitionUnitsList/AcquisitionUnitsList.js [hunk 2]
```diff
@@ -29,25 +31,38 @@ const AcquisitionUnitsList = ({ acquisitionUnits, getViewPath, getCreatePath })
     history.push(getViewPath(id))
   ), [getViewPath, history]);
 
+  const shortcuts = [
+    {
+      name: 'new',
+      handler: () => history.push(getCreatePath()),
```
Comment (AndreiShumski2) [inline src/settings/AcquisitionUnits/AcquisitionUnitsList/AcquisitionUnitsList.js:37]: Please, check permissions for create/edit

File: src/settings/AcquisitionUnits/AcquisitionUnitDetails/AcquisitionUnitDetails.js (hunk 1 above)
Comment (usavkov-epam) [reply src/settings/AcquisitionUnits/AcquisitionUnitDetails/AcquisitionUnitDetails.js]: updated

File: src/settings/AcquisitionUnits/AcquisitionUnitsList/AcquisitionUnitsList.js (hunk 2 above)
Comment (usavkov-epam) [reply src/settings/AcquisitionUnits/AcquisitionUnitsList/AcquisitionUnitsList.js]: updated

File: src/settings/AcquisitionUnits/AcquisitionUnits.js [hunk 3]
```diff
@@ -17,51 +22,55 @@ const AcquisitionUnits = ({ match, history }) => {
...
-          </Fragment>
-        )}
-      />
-    </Switch>
+          )}
+        />
+        <Route
+          path={path}
+          render={() => (
+            <Fragment>
```
Comment (AndreiShumski2) [inline src/settings/AcquisitionUnits/AcquisitionUnits.js:52]: `<>`

**Review by AndreiShumski2 — APPROVED** [review]

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**